packages/web3-constants/evm/rpc.json

```diff
@@ -83,7 +83,7 @@
         "Rinkeby": ["https://rinkeby.infura.io/v3/d65858b010d249419cf8687eca12b094"],
         "Kovan": ["https://kovan.infura.io/v3/d65858b010d249419cf8687eca12b094"],
         "Gorli": ["https://goerli.infura.io/v3/d65858b010d249419cf8687eca12b094"],
-        "BSC": ["https://bsc-dataseed1.binance.org"],
```

Comment:
Why? This setting copy is from https://chainlist.wtf. MetaMask will raise an error if RPC URLs do not match.

<img width="429" alt="image" src="https://user-images.githubusercontent.com/52657989/212267234-9583b52e-3703-4ef0-894b-6fcfc8e28906.png">

Reply:
Fortmatic complains about `https://bsc-dataseed1.binance.org`.

OK. I can use rpc url from `RPC_URLS` instead, it would satisfy both MetaMask and Fortmatic.